processor/otel/consumer.go

```diff
@@ -408,6 +418,71 @@ func parseSpan(span *tracepb.Span, event *model_span.Event) {
 	event.Labels = labels
 }
 
+func parseErrors(source string, otelSpan *tracepb.Span) []*model_error.Event {
+	var errors []*model_error.Event
+	for _, log := range otelSpan.GetTimeEvents().GetTimeEvent() {
+		var isError bool
+		var logMessage, exMessage string
+		for k, v := range log.GetAnnotation().GetAttributes().GetAttributeMap() {
+			if source == sourceFormatJaeger {
+				switch v := v.Value.(type) {
+				case *tracepb.AttributeValue_StringValue:
+					vStr := v.StringValue.Value
+					switch k {
+					case "event":
+						logMessage = vStr
+					case "level":
+						isError = vStr == "error"
+					case "error":
+						exMessage = vStr
```

Comment:
should this also set `isError`? or is `level` always present?

Reply:
Maybe also "error.kind" for exception.type?

Reply:
In the examples I saw it was always present, but doesn't hurt to also set in this case.